### DISCUSSION TITLE
Reports Function and Redis

### DISCUSSION THREAD

**Discussion by tohtohro** [netbox-community/netbox#5898] [Q&A] [answered]
Just to clarify - is redis a requirement for the reports function to work? 
If not, how do I go about using reports without redis?

**Comment by candlerb** [comment, marked as answer]
These days, redis is a hard requirement for Netbox to work at all.  It became mandatory as of [v2.6.0](https://github.com/netbox-community/netbox/releases/v2.6.0)
